### PR TITLE
Add io.github.dvlv.boxbuddyrs

### PR DESCRIPTION
**Pull request by Dvlv**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*

**BoxBuddy needs `flatpak-spawn` access to use the host's Distrobox. A PR was opened here:** https://github.com/flathub/flatpak-builder-lint/pull/230

- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.dvlv.boxbuddyrs.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+         "*.sh"
+      ],
+      "sources": [{
+         "type": "git",
+         "url": "https://github.com/Dvlv/BoxBuddyRS",
+         "tag": "1.0.4",
+         "builddir": true,
+	 "commit": "2e3a77f6a236da234ee220c10fbd0f578c42f8df"
+      },
+          "generated-sources.json"
```
Comment (hfiguiere) [inline io.github.dvlv.boxbuddyrs.json:56]: that file is missing.

Comment (Dvlv) [reply io.github.dvlv.boxbuddyrs.json:56]: Sorry, added that file.

**Comment by barthalion** [comment]
bot, build io.github.dvlv.boxbuddyrs

**Pull request closed without merging**